common/i18n/resources/en.js

```diff
@@ -1,11 +1,40 @@
 module.exports = {
+  common: {
+    name: 'name',
+    cancel: 'cancel',
+    delete: 'delete',
+    cantUndoWarning: 'This operation cannot be undone!',
```

Comment:
Let's put this key where it's used. We'll move it to general, when it'll be used more than once?

Reply:
I checked, they are used more than once but haven't gone yet thought the other modules where they are used